py/setup.py

```diff
@@ -239,7 +239,7 @@ def run(self):
         libraries=["torchtrt"],
         include_dirs=[
             dir_path + "torch_tensorrt/csrc", dir_path + "torch_tensorrt/include",
-            dir_path + "/../bazel-TRTorch/external/tensorrt/include",
```

Comment:
We want to keep this, maybe try reordering

Reply:
fixed it. Let me find a workaround for this.